audit_scope.md

```diff
@@ -0,0 +1,32 @@
+## VVV Venture Capital Audit Scope
+
+1. **Scope** 
```

Comment:
This would make sense, given it's use in the current contracts. Glad to include if you prefer.

I assumed since it was in-scope in the [last audit](https://docs.google.com/document/d/10_t7Kt814Otp-FMFK8mvCsxb3tX3wyu1Z9V4nhZxTY8/edit?tab=t.0#heading=h.por8yrqqn8f6) that it was not needed.